### DISCUSSION TITLE
域名合法性校验失败,当前环境已有对应域名证书,不可重复申请

### DISCUSSION THREAD

**Discussion by KimmyLueng** [FranzKafkaYu/x-ui#205] [Q&A]
要怎样找出证书文件位置，并删除

**Comment by KimmyLueng** [comment]
找到了

**Reply by wayneace** [reply]
我把cert文件夹整个删掉了还是提示这个错误，请问还需要删除什么文件？

**Reply by KatsuraNTR** [reply]
> 我把cert文件夹整个删掉了还是提示这个错误，请问还需要删除什么文件？

acme.sh  --remove  -d example.com
